build.gradle

```diff
@@ -1,225 +1,99 @@
 
-////////////////////////////////////////////////////////////////////////////////////////////////////
-// Maven-publish - Configuration for publishing maven artifacts
-////////////////////////////////////////////////////////////////////////////////////////////////////
-publishing {
-    // List of artifacts to publish
-    publications {
+        inputs.files("$patchingScript", "$graalPatch")
+        outputs.files("$graalCloneDummy")
 
-        // Publish the tar file that contains graal
-        maven(MavenPublication) {
+        doFirst {
+            mkdir tempDir
+            workingDir tempDir
+            commandLine "$patchingScript", "$graal_version", "$graal_commit_id"
+            //commandLine "ls"
 
-            // Artifact to be publish
-            artifact(graalVMTar) {
-                extension 'tar.gz'
-                builtBy(tarGraal)
-            }
-
-            // Details about the artifact being publish
-            groupId = "com.r3.conclave"
-            artifactId = "graalvm"
-            version = "$graal_version-$conclave_version"
-            description = 'Graal distribution including SubstrateVM and tools'
+            // Write current date to graal clone dummy file. If this task is run again, the content of this file
+            // will change and subsequent tasks will run.
+            writeDateToFile(graalCloneDummy)
         }
     }
 
-    // List of repositories where the artifact is published
-    repositories {
-
-        // Artifactory repository
-        maven {
-            name = "artifactory"
-            url = "${environment.OBLIVIUM_MAVEN_URL}/${environment.OBLIVIUM_MAVEN_REPOSITORY}"
-            credentials {
-                username = environment.OBLIVIUM_MAVEN_USERNAME
-                password = environment.OBLIVIUM_MAVEN_PASSWORD
+    ////////////////////////////////////////////////////////////////////////////////////////////////////
+    // Maven-publish - Configuration for publishing maven artifacts
+    ////////////////////////////////////////////////////////////////////////////////////////////////////
+    publishing {
+        // List of repositories where the artifact is published
+        repositories {
+
+            // Artifactory repository
+            maven {
+                name = "artifactory"
+                url = "${environment.OBLIVIUM_MAVEN_URL}/${environment.OBLIVIUM_MAVEN_REPOSITORY}"
+                credentials {
+                    username = environment.OBLIVIUM_MAVEN_USERNAME
+                    password = environment.OBLIVIUM_MAVEN_PASSWORD
+                }
             }
-        }
 
-        // Local repository - Useful for testing purposes
-        maven {
-            name = "LocalRepo"
-            url = "$buildDir/repo/graal-vm"
+            // Local repository - Useful for testing purposes
+            maven {
+                name = "LocalRepo"
+                url = "$buildDir/repo/graalVM"
```

Comment:
The README doesn't mention the `graalVM` subdirectory, but I don;t even think it's needed. In fact, isn't it incorrect to use `graalVM` since it applies to all artifacts, including the SDK?